front/src/styles/scss/applications/stdcmV2/_home.scss

```diff
@@ -29,15 +29,35 @@
                 /*TODO Waiting to fix the button in ui-core...*/
                 .stdcm-v2-launch-request{
                     padding-bottom: 0.15rem;
+
                     button {
                         justify-content: center;
                         width: 100%;
                         font-weight: 500;
                     }
                 } 
+                .simulation-availaible {
+                  width: 34.125rem;
+                  height: 5rem;
+                  display: flex;
+                  flex-direction: column;
+                  position: absolute;
+                  margin-top: 2.5rem;
+                  border-radius: 0.5rem;
+                  background-color: rgb(230, 247, 238);
+                  box-shadow:  0px 0px 0px 4px rgba(255, 255, 255, 1) inset, 0px 0px 0px 5px rgb(60, 202, 128) inset, 0px 3px 7px -3px rgba(11, 114, 60, 0.4);
+                  
+                    span {
```

Comment:
It seems like indentation is still a bit off here. You can automatically format the file with this command:

```
yarn prettier --write front/src/styles/scss/applications/stdcmV2/_home.scss
```

Reply:
thank you !